sunpy/net/dataretriever/sources/goes.py

```diff
@@ -243,17 +242,22 @@ def _get_url_for_timerange(self, timerange, **kwargs):
         base_url = "https://data.ngdc.noaa.gov/platforms/solar-space-observing-satellites/goes/goes{goes_number}/"
         supported_waves = [94, 131, 171, 195, 284, 304]
         supported_levels = ("2", "1b")
+        min_satellite_number = 16  # when SUVI was first included
 
         # these are optional requirements so if not provided assume defaults
         # if wavelength is not provided assuming all of them
         if "wavelength" in kwargs.keys():
+            if int(kwargs.get("wavelength").to_value('Angstrom')) not in supported_waves:
```

Comment:
can you test what happens if wavelength is a range. i.e `a.Wavelength(90*u.AA, 150*u.AA)` should get both 94 and 131. I don't think this will work like that though.

Reply:
How does this range get passed to `_get_url_for_timerange`?

Reply:
see above.